datasources/barters.js

```diff
@@ -1,44 +1,111 @@
-const ItemsAPI = require('./items');
-const itemsAPI = new ItemsAPI();
-
 class BartersAPI {
-  async getList() {
-    const barters = await ITEM_DATA.get('BARTER_DATA', 'json');
-
-    if(!barters){
-        return {};
+    constructor(){
+        this.caache = false;
```

Comment:
```suggestion
        this.cache = false;
```

Another typo perhaps? 😉

Reply:
Fixed also!